File: Dialog/app/src/main/java/com/example/dialog/MainActivity.java
package com.example.dialog;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Display;
import android.view.View;
import android.view.WindowManager;

import com.example.dialog.databinding.ActivityMainBinding;
import com.liys.dialoglib.LDialog;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        WindowManager windowManager = getWindowManager();
        Display display = windowManager.getDefaultDisplay();
        binding.button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                LDialog dialog = LDialog.newInstance( MainActivity.this,R.layout.dialog);
                dialog
                        .setWidth(display.getWidth())
                        .setHeight(display.getHeight())

                        .show();
            }
        });


        binding.textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                binding.textView.setText("123");
            }
        });



    }


}

File: JHWeatherKotlin/app/src/main/java/com/example/jhweatherkotlin/MainActivity.kt
package com.example.jhweatherkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import androidx.lifecycle.ViewModelProvider
import com.example.jhweatherkotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        var weatherViewModel : WeatherViewModel = ViewModelProvider(this).get(WeatherViewModel::class.java)

        var binding : ActivityMainBinding = DataBindingUtil.setContentView(this,R.layout.activity_main)

        binding.lifecycleOwner = this


    }
}